### DISCUSSION TITLE
Unlinked stacking/tabbing

### DISCUSSION THREAD

**Discussion by vbartle** [i3/i3#5972] [Ideas]
Is it possible to conduct stacking/tabbing such that I don't have to conjure up a plan well ahead of time? 

For example, if I want to move a window vertically next to the other, I have to set the other at +v, then move the initial window toward it. Or for example, if I have a vertical split, and I want to tab the bottom window, I need to first make a horizontal split, then switch it to tabbed. 

Can someone explain why not have a command that immediately converts the current window into a tabbed layout? I.e. desired functionality would be mod+[something] which results in a directly tabbed window for the bottom of a vertically split window pane. Is there actually a way to do this that I missed in the docs?

**Comment by okraits** [comment]
As I understand it: you can only change the layout of split windows, thus you have to split a single window before you can change its layout.
This is explained in depth in i3's userguide and in the video about containers and the tree data structure.

Using [Command chaining](https://i3wm.org/docs/userguide.html#command_chaining) you can create a keybinding which splits the window and changes the layout to tabbed in one go:

`bindsym $mod+t splitv, layout tabbed`

**Reply by vbartle** [reply]
Thank you, yeah the mechanics make sense just the end action feels clunky. Which I guess is what command chains help with!
